docs/user-guide/resource-browser.md

```diff
@@ -1,71 +1,438 @@
+    Devtron ignores the 'command' field while launching an ephemeral container
+    {% endhint %}
+
+---
+
+## Other Resource Kinds
+
+Other resources in the cluster are grouped under the following categories:
+
+* Namespace
+* Workloads
+* Config & Storage
+* Networking
+* RBAC
+* Administration
+* Other Resources
+* Custom Resource
+
+![Figure 18: Resources within Cluster](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/resource-list.jpg)
+
+---
+
+## Cluster Terminal
+
+User with [super-admin](./global-configurations/authorization/user-access.md#assign-super-admin-permission) access can now troubleshoot cluster issues by accessing the cluster terminal from Devtron. You can select an image from the list that has all CLI utilities like kubectl, helm, netshoot etc. or can use a custom image, which is publicly available.
+
+To troubleshoot a cluster or a specific node in a cluster, click the terminal icon on the right side.
+
+![Figure 19: Terminal Icon](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/cluster-terminal.gif)
+
+* You will see the user-defined name for the cluster in Devtron. E.g. `default-cluster`.
+* Select the node you wish to troubleshoot from the `Node` drop-down. E.g. `demo-new`.
+* Select the namespace from the drop-down list which you have added in the [Environment](./global-configurations/cluster-and-environments.md#add-environment) section.
+* Select the image from the drop-down list which includes all CLI utilities or you can use a custom image, which is publicly available.
+* Select the terminal shell from the drop-down list (e.g. `sh`, `bash`) to troubleshoot a node.
+
+### Use Case - Debugging Pods
+
+You can also create pod for debugging which will connect to pod terminal. To find out why a particular pod is not running, you can check `Pod Events` and `Pod Manifest` for details.
+
+The **Debug Mode** is helpful in scenarios where you can't access your Node by using an SSH connection. When enabled, a pod is created on the node, which opens an interactive shell on the Node.
+
+* Check the current state of the pod and recent events with the following command:
 
 ```bash
+kubectl get pods
+```
+
+* To know more information about each of these pods and to debug a pod depending on the state of the pods, run the following command:
+
+```bash
+kubectl describe pod <podname>
+```
+
+| Pod Status       | Description                             | Common Causes                              |
+| ---------------- | --------------------------------------- | ------------------------------------------ |
+| Pending          | Pod cannot be scheduled onto a node     | Insufficient resources (CPU, memory, etc.) |
+| Waiting          | Pod has been scheduled but cannot run   | Failed to pull container image             |
+| CrashLoopBackOff | Container keeps crashing and restarting | Incorrect access token for API interaction |
```

Comment:
Is this example pod statuses or all pod statuses? [Ref](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/)

Reply:
Those were example pod statuses. But, I get what you're saying, the list of statuses and conditions is exhaustive. 
Let me just remove the statuses and point them to the K8s reference instead.